main.spec

```diff
@@ -0,0 +1,37 @@
+# -*- mode: python ; coding: utf-8 -*-
```

Comment:
please add a comment briefly describing what we're doing here and what special steps you needed to take